internal/handlers/common/projection.go

```diff
@@ -32,33 +34,33 @@ func isProjectionInclusion(projection *types.Document) (inclusion bool, err erro
 		case bool:
 			if v {
 				if exclusion {
-					err = lazyerrors.New(errMsg)
```

Comment:
That's exactly why there is no XXXf function for errors – `errMsg` is not a template for printf-like function, that's a message. Using messages where templates are expected is one of the common sources of security problems. For example, consider what would happen if the message becomes "projection must be 100% inclusions or exclusions".

In short, it should be either `fmt.Errorf("%s", errMsg)` or `errors.New(errMsg)`

Reply:
now see.